DevDocumentation/gnu-make.md

```diff
@@ -0,0 +1,92 @@
+A quick primer on GNU Make syntax
+=================================
+
+This tries to cover the syntax that is hard to ctrl-f for in
+<https://www.gnu.org/software/make/manual/make.html> (err, hard to
+C-s for in `M-: (info "Make")`).
+
+  At the core is a "rule":
+
+      target: dependency1 dependency2
+      	command to run
+
+  If `target` something that isn't a real file (like 'build', 'lint', or
+  'test'), then it should be marked as "phony":
+
+      target: dependency1 dependency2
+      	command to run
+      .PHONY: target
+
+  You can write reusable "pattern" rules:
+
+      %.o: %.c
+      	command to run
+
+  Of course, if you don't have variables for the inputs and outputs,
+  it's hard to write a "command to run" for a pattern rule.  The
+  variables that you should know are:
+
+      $@ = the target
+      $^ = the list of dependencies (space separated)
+      $< = the first (left-most) dependency
+      $* = the value of the % glob in a pattern rule
+
+      Each of these have $(@D) and $(@F) variants that are the
+      directory-part and file-part of each value, respectively.
+
+      I think those are easy enough to remember mnemonically:
+        - $@ is where you shoul direct the output at.
```

Comment:
`s/shoul/should/` when you get a chance.

Reply:
Fixed.